app/2.1.x/hybrid-mode.md

```diff
@@ -27,6 +27,31 @@ Deploying using Hybrid mode has a number of benefits:
 * Easiness of management, since an admin will only need to interact with the CP nodes to control
   and monitor the status of the entire Kong cluster.
 
+## Configuration Properties
+
+Hybrid Mode introduces the following configuration properties:
+- `cluster_listen`: List of addresses and ports on which the Control Plane will listen
+for incoming Data Plane connections. Defaults to `0.0.0.0:8005`.
+- `cluster_control_plane`: Port on which the Data Plane nodes will attempt to connect
+to the Control Plane. Must contain the `cluster_listen` value that was configured in
+the Control Plane node. *Required*.
+- `cluster_mtls`: One of `shared` or `pki`. Indicates whether Hybrid Mode will use a
+shared certificate/key pair for CP/DP mTLS or if PKI mode will be used.  If set to
+`shared`, `cluster_cert` and `cluster_cert_key` must be configured and specify the
+same certificate on CP and DP nodes.  If set to `pki`, these should specify different
+node certificates, but a common CA certificate should be presented via the
+ additional `cluster_cert` and `cluster_server_name` settings. Defaults to `shared`.
+- `cluster_cert` and `cluster_cert_key`: Certificate/Key pair used for mTLS between
+CP/DP nodes.  Under `shared` mode should be the same certificate for CP/DP nodes,
+under `pki` mode they should be unique per node, but generated from the same CA.
+*Required*.
+- `cluster_ca_cert`: The trusted CA certificate file in PEM format used to verify
+the `cluster_cert`. *Required* when `cluster_mtls` is set to `pki`, *ignored* otherwise.
```

Comment:
```suggestion
- `cluster_ca_cert`: The trusted CA certificate file used to verify the 
`cluster_cert`, in PEM format. *Required* when `cluster_mtls` is set to `pki`, 
*ignored* otherwise.
```